src/Thumbnail/FileThumbnail.php

```diff
@@ -0,0 +1,58 @@
+<?php
+
+declare(strict_types=1);
+
+/*
+ * This file is part of the Sonata Project package.
+ *
+ * (c) Thomas Rabaix <thomas.rabaix@sonata-project.org>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Sonata\MediaBundle\Thumbnail;
+
+use Sonata\MediaBundle\Model\MediaInterface;
+use Sonata\MediaBundle\Provider\MediaProviderInterface;
+use Symfony\Component\Asset\Packages;
+
+/**
+ * @author Jordi Sala Morales <jordism91@gmail.com>
+ */
+final class FileThumbnail implements ThumbnailInterface
+{
+    /**
+     * @var Packages
+     */
+    private $packages;
+
+    public function __construct(Packages $packages)
+    {
+        $this->packages = $packages;
+    }
+
+    public function generatePublicUrl(MediaProviderInterface $provider, MediaInterface $media, string $format): string
+    {
+        if (MediaProviderInterface::FORMAT_ADMIN !== $format) {
+            throw new \RuntimeException(sprintf('Unable to generate thumbnail for the given format %s.', $format));
+        }
+
+        return $this->packages->getUrl('bundles/sonatamedia/file.png');
+    }
+
+    public function generatePrivateUrl(MediaProviderInterface $provider, MediaInterface $media, string $format): string
+    {
+        throw new \RuntimeException('Unable to generate private thumbnail url for media files.');
+    }
+
+    public function generate(MediaProviderInterface $provider, MediaInterface $media): void
+    {
+        // nothing to generate, as the thumbnails for files are already generated
+    }
+
+    public function delete(MediaProviderInterface $provider, MediaInterface $media, $formats = null): void
+    {
+        // feature not available
```

Comment:
Should we throw an exception ?

Reply:
I copied another thumbnail implementation. Should I change it too?

Reply:
Honestly I dunno if we should throw a method not implemented exception or doing nothing becase some manager could use the delete method every time and rely on the fact the feature is really removed on not.

I would say the best solution would be to remove the `delete` method from the interface and add a DeletableThumbnailInterface for this. This way you can remove the delete method here.

Do you want to do this here, in another PR or never ? ^^

Reply:
I tried to do it in this PR, wdyt?